src/utils/wallet.py

```diff
@@ -0,0 +1,1473 @@
+"""This submodule provides the PublicKey, PrivateKey, and Signature classes.
+It also provides HDPublicKey and HDPrivateKey classes for working with HD
```

Comment:
is this an open source bit of code?  if so, maybe its worth using `pip` or at least putting attribution in it?

Reply:
Good catch, yes it open source. I forgot to add the attribution. I will see if it is available via pip or if something similar is.